app/controllers/api/v3/subject_areas_controller.rb

```diff
@@ -0,0 +1,9 @@
+module API
+  module V3
+    class SubjectAreasController < API::V3::ApplicationController
+      def index
+        render jsonapi: SubjectArea.active, fields: fields_param, include: params[:include], class: CourseSerializersService.new.execute[:v3]
```

Comment:
Is the idea we'll just use `include: :subjects` to get the whole chain in one request?

Reply:
Yes